streaming/src/main/scala/org/apache/carbondata/streaming/parser/FieldConverter.scala

```diff
@@ -63,16 +66,22 @@ object FieldConverter {
         case b: java.lang.Boolean => b.toString
         case s: java.lang.Short => s.toString
         case f: java.lang.Float => f.toString
-        case bs: Array[Byte] => new String(bs,
-          Charset.forName(CarbonCommonConstants.DEFAULT_CHARSET))
+        case bs: Array[Byte] =>
+          if (isInsertFlow) {
+            Base64.getEncoder.encodeToString(bs)
```

Comment:
could you pls add some comment here about the reason in case 'isInsertFlow' use base64

Reply:
OK

Reply:
done